src/main/java/graphql/GraphQL.java

```diff
@@ -279,7 +279,7 @@ public static class Builder {
         private DataFetcherExceptionHandler defaultExceptionHandler = new SimpleDataFetcherExceptionHandler();
         private ExecutionIdProvider idProvider = DEFAULT_EXECUTION_ID_PROVIDER;
         private Instrumentation instrumentation = null; // deliberate default here
-        private PreparsedDocumentProvider preparsedDocumentProvider = NoOpPreparsedDocumentProvider.INSTANCE;
+        private PreparsedDocumentProvider preparsedDocumentProvider = new CachingDocumentProvider();
```

Comment:
caching is now on by default.  If you dont want it use `NoOpPreparsedDocumentProvider.INSTANCE` which still exists